presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -86,6 +87,7 @@
         implements HiveMetastore
 {
     protected final HiveMetastore delegate;
+    private final boolean impersonationEnabled;
```

Comment:
We can remove this field if we add `isImpersonationEnabled()` to `HiveMetastore`. Then `updateIdentity()` can simply call `delegate.isImpersonationEnabled()`.

Reply:
Should we always return `false` in FileHiveMetastore and GlueHiveMetastore?

Reply:
Yes, always false. we don't support impersonation there